mods/farming/api.lua

```diff
@@ -83,10 +83,30 @@ farming.register_hoe = function(name, def)
 		end
 	})
 	-- Register its recipe
-	minetest.register_craft({
-		output = name:gsub(":", "", 1),
```

Comment:
I just used the existing code. I've never used  gsub before and honestly have no clue what it does. Give me some time to look it up and fix this.